.github/workflows/azwi-e2e.yaml

```diff
@@ -55,16 +49,15 @@ jobs:
           echo "SERVICE_ACCOUNT_NAME=${SERVICE_ACCOUNT_NAMESPACE}-sa" >> "${GITHUB_ENV}"
       - name: Create kind cluster
         run: |
-          # create a kind cluster with predefined signing keys
-          echo ${{ secrets.SERVICE_ACCOUNT_SIGNING_KEY }} | base64 -d > sa.key
-          echo ${{ secrets.SERVICE_ACCOUNT_KEY }} | base64 -d > sa.pub
+          openssl genrsa -out sa.key 2048
+          openssl rsa -in sa.key -pubout -out sa.pub
           make kind-create
       - name: Build azwi
         run: make bin/azwi
       - uses: azure/login@v1.4.3
```

Comment:
Looks like `ACTIONS_ID_TOKEN_REQUEST_URL` and `ACTIONS_ID_TOKEN_REQUEST_TOKEN` won't be available to this action until it's merged to the main branch, which is why azure cli login failed. Let's ignore and I will open a follow-up PR to fix the CI failure.